sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/aio/_azure_appconfiguration_client_async.py

```diff
@@ -14,25 +14,20 @@
 from azure.core.polling import AsyncLROPoller
 from azure.core.tracing.decorator import distributed_trace
 from azure.core.tracing.decorator_async import distributed_trace_async
-from azure.core.exceptions import (
-    ResourceExistsError,
-    ResourceModifiedError,
-    ResourceNotFoundError,
-    ResourceNotModifiedError,
-)
+from azure.core.exceptions import ResourceNotModifiedError
 from azure.core.rest import AsyncHttpResponse, HttpRequest
 from ._sync_token_async import AsyncSyncTokenPolicy
 from .._azure_appconfiguration_error import ResourceReadOnlyError
 from .._azure_appconfiguration_requests import AppConfigRequestsCredentialsPolicy
-from .._generated.aio import AzureAppConfiguration
+from .._generated.aio import AzureAppConfigurationClient as AzureAppConfigurationClientGenerated
 from .._generated.models import (
-    SnapshotUpdateParameters,
     SnapshotStatus,
     SnapshotFields,
     SnapshotComposition,
     LabelFields,
     ConfigurationSettingFields,
 )
+from .._generated.models._models import SnapshotUpdateParameters
```

Comment:
I guess you should update client.tsp to expose it?

Reply:
This is a good question, I need to check tsp.

Reply:
Updated